setup.py

```diff
@@ -89,6 +89,11 @@ def get_extensions() -> List:
     sources = (
         list(glob.glob(os.path.join(extensions_dir, "*.cpp"))) +
         list(glob.glob(os.path.join(extensions_dir, "*.cu"))))
+    
```

Comment:
minor, could we put them inside `include_dirs` and `sources` definition? (Looks slightly cleaner to me)